packages/icons/src/index.js

```diff
@@ -123,6 +123,7 @@ export { default as moreHorizontalMobile } from './library/more-horizontal-mobil
 export { default as moreVertical } from './library/more-vertical';
 export { default as moveTo } from './library/move-to';
 export { default as navigation } from './library/navigation';
+export { default as none } from './library/none';
```

Comment:
Alternatively this could be named for the icon form, rather than it's purpose, e.g. `shortDash`. Any recommendations @jasmussen @jameskoster?

<img width="99" alt="image" src="https://user-images.githubusercontent.com/1699996/118017880-bc078e00-b31c-11eb-88e0-1bee5aa432d6.png">

Reply:
Is this the same icon we're using the the `solid` border style? cc @aaronrobertshaw :D 

This could be seen as an antonym to the "Add" icon (+). So perhaps "Remove", "Subtract", "Minus" ?

Reply:
I took the remove icon from [Icons / Editor](https://www.figma.com/file/e4tLacmlPuZV47l7901FEs/WordPress-Design-Library?node-id=5415%3A22292) (it's called [`minus` in the icons package](https://github.com/WordPress/gutenberg/blob/0ae6aee0bed7bdd579223b211538cd7131767c75/packages/icons/src/library/minus.js)) and shortened it from 12 px to 8 px. The minus icon seemed a little wide to me, as is.

Here's a [figma link](https://www.figma.com/file/Ngtfpl2j52PJksXJzoO9EuPo/None-Icon-for-Gutenberg?node-id=0%3A1).